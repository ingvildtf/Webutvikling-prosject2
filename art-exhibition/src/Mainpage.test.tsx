import React from 'react'
import renderer from 'react-test-renderer'
import Mainpage from './mainpage'

test('First snapshot test', () => {
  const component = renderer.create(<Mainpage />)
  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
