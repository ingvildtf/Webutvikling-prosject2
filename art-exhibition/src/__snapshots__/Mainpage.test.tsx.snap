// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First snapshot test 1`] = `
Array [
  " ",
  <div
    className="sc-cxNIbT cnhoeV"
  >
    <div
      className="sc-lmgQde eIXylf"
    >
      <div
        className="sc-bdnylx eHEgeA"
      >
        <div
          className="sc-gtssRu kmgSDO navbar-container container"
        >
          <a
            className="logo"
            href="/"
            onClick={[Function]}
          >
            SPACE ART
          </a>
          <a
            className="link"
            href="/my-favorites"
            onClick={[Function]}
          >
            My Favorites
          </a>
        </div>
      </div>
    </div>
    <div
      className="sc-lmgQde cpoeXm"
    >
      <div
        className="sc-hBMVcZ ftcjow"
        style={
          Object {
            "background": "#0B1226",
            "color": "white",
          }
        }
      >
        <h3
          className="sc-fnVYJo eoqoju"
          style={
            Object {
              "background": "#0B1226",
              "color": "white",
            }
          }
        >
          Earth
        </h3>
        <div
          className="sc-iemXMA hRgXLv"
        >
          <div>
            <svg
              className="sc-dlnjPT kipuPJ"
            >
              earth.svg
            </svg>
          </div>
        </div>
        <div
          className="sc-fFSRdu jTLrLq"
          style={
            Object {
              "background": "#0B1226",
              "color": "white",
            }
          }
        >
          <div>
            <br />
          </div>
        </div>
        <div
          className="sc-bkbjAj kFhcbs"
        >
          <div>
            <audio
              controls={true}
              loop={true}
              src="earth.mp3"
            />
          </div>
        </div>
        <div
          className="sc-dIvqjp kFlQOd"
          style={
            Object {
              "background": "#0B1226",
              "color": "white",
            }
          }
        >
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Back
          </button>
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Next
          </button>
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Favoritt
          </button>
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Space
          </button>
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Easter
          </button>
          <button
            className="sc-hHEjAm SmlJl"
            onClick={[Function]}
            style={
              Object {
                "background": "#54728C",
                "color": "white",
              }
            }
          >
            Xmas
          </button>
        </div>
      </div>
    </div>
  </div>,
]
`;
